Clarify Fabric Private Link connection creation steps (#9705)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/secure/private-connectivity/azure/fabric.md

Once dbt Support confirms the endpoint is available, configure the connection in <Constant name="dbt" />:

1. Navigate to **Settings** → **Create new connection** → select **Microsoft Fabric**.
2. You will see two radio buttons: **Default Endpoint** and **PrivateLink Endpoint**. Select **PrivateLink Endpoint**.
3. Select the private endpoint from the dropdown. This populates the **Server** field with the private SQL hostname provided by dbt Support.
4. Configure the remaining connection details:

   <SimpleTable> 
   | Field | Value |
   |-------|-------|
   | **Port** | Auto-populated with `1433` |
   | **Database** | Your Fabric warehouse name |
  </SimpleTable>
  

> 5. Test your connection and save it.

Reviewer: I don't think you can test the connection at this stage (there are no credentials yet). All you can do is save, then proceed to the next step.

> 5. Test your connection and save it.

Author: ```suggestion
5. Save it and proceed to the next step.
```

Once dbt Support confirms the endpoint is available, configure the connection in <Constant name="dbt" />:

1. Navigate to **Settings** → **Create new connection** → select **Microsoft Fabric**.
2. You will see two radio buttons: **Default Endpoint** and **PrivateLink Endpoint**. Select **PrivateLink Endpoint**.
3. Select the private endpoint from the dropdown. This populates the **Server** field with the private SQL hostname provided by dbt Support.
4. Configure the remaining connection details:

<SimpleTable> 
   

> | Field | Value |

Reviewer 2: I suggest we indent all lines for the table so it'll be under Step 4.

Author: i trried but it was given me a vecel error

   |-------|-------|
   | **Port** | Auto-populated with `1433` |
   | **Database** | Your Fabric warehouse name |

>   </SimpleTable>

Author: ```suggestion
  </ SimpleTable>
```

> 6. Save it and proceed to the next step.

Author: ```suggestion
5. Save it and proceed to the next step.
```

### Configure authentication

> The **Connection** page only includes **Server**, **Port**, and **Database**. You configure the following later when you set up your development credentials or create a deploy environment profile:

Reviewer 2: I suggest we add links to instructions on how to set up dev credentials/create a deploy environment profile.
```suggestion
The **Connection** page only includes **Server**, **Port**, and **Database**. Configure the following later when you set up your development credentials or create a deploy environment profile:
```

> - **Authentication**: This is your Service Principal

Reviewer 2: Can we also present this as a simple table for consistency? The Value description can then be shortened to "Your Microsoft Entra.."

- **Tenant ID**: This is your Microsoft Entra Directory (tenant) ID
- **Client ID**: This is your service principal's application (client) ID
- **Client secret**: This is your service principal's client secret

> The service principal you use for authentication must be added as a **Member** or **Contributor** on the Fabric workspace. In the Fabric Admin portal, **Service principals can use Fabric APIs** must also be enabled under **Tenant settings** → **Developer settings**.

Reviewer 2: Minor editorial suggestions to make this active:
```suggestion
Add the service principal you use for authentication as a **Member** or **Contributor** on the Fabric workspace. In the Fabric Admin portal, you must also enable **Service principals can use Fabric APIs** under **Tenant settings** → **Developer settings**.
```

Once dbt Support confirms the endpoint is available, configure the connection in <Constant name="dbt" />:

1. Navigate to **Settings** → **Create new connection** → select **Microsoft Fabric**.
2. You will see two radio buttons: **Default Endpoint** and **PrivateLink Endpoint**. Select **PrivateLink Endpoint**.
3. Select the private endpoint from the dropdown. This populates the **Server** field with the private SQL hostname provided by dbt Support.
4. Configure the remaining connection details:

    <SimpleTable> 
       
    | Field | Value |
    |-------|-------|
    | **Port** | Auto-populated with `1433` |
    | **Database** | Your Fabric warehouse name |
    
    </ SimpleTable>
  
5. Save it and proceed to the next step.

### Configure authentication

> The **Connection** page only includes **Server**, **Port**, and **Database**. Configure the following later when you set up your development credentials or create a deploy environment profile:

Reviewer 2: If applicable:
```suggestion
The **Connection** page only includes **Server**, **Port**, and **Database**. Configure the following later when you [set up your development credentials](/docs/dbt-platform-environments#set-user-credentials) or [create a deployment environment](/docs/deploy/deploy-environments):
```
